ios/Classes/FacebookModule.m

```diff
@@ -413,21 +343,18 @@ -(void)authorize:(id)args
 -(void)initialize:(id)args
 {
     ENSURE_SINGLE_ARG_OR_NIL(args, NSNumber);
-    double timeoutInMs = [TiUtils intValue:args def:0];
-    if (timeoutInMs == 0) {
-        meRequestTimeout = 180.0;
-    } else {
-        meRequestTimeout = (double)timeoutInMs / 1000.0;
+    if (args != nil) {
+        DEPRECATED_REMOVED(@"timeout param", @"5.0.0", @"5.0.0");
```

Comment:
Considering, if "timeout param" is the right name to conform with the log output.